rtc-video-quality/generate_data.py

```diff
@@ -461,8 +463,11 @@ def generate_jobs(args, temp_dir):
                 job_temp_dir = tempfile.mkdtemp(dir=temp_dir)
                 (command, encoded_files) = get_encoder_command(job['encoder'])(
                     job, job_temp_dir)
-                command[0] = find_absolute_path(args.use_system_path,
-                                                command[0])
+                full_command = find_absolute_path(args.use_system_path,
+                                                  command[0])
+                command = [
+                    word.replace(command[0], full_command) for word in command
```

Comment:
It doesn't make much sense to do a substring replacement here. I'd think something like this would be more appropriate: 

[full_command for word in command if word == command[0] else word]

Reply:
Oh right this makes sense